src/Powercord/coremods/settings/index.js

```diff
@@ -47,14 +47,27 @@ async function patchSettingsComponent () {
 
     const changelog = sections.find(c => c.section === 'changelog');
     if (changelog) {
-      const settingsSections = Object.keys(powercord.api.settings.tabs).map(s => _makeSection(s));
+      const coreSections = Object.keys(powercord.api.settings.tabs)
+        .filter(s => ['pc-moduleManager-themes', 'pc-moduleManager-css', 'pc-moduleManager-plugins', 'pc-general', 'pc-updater'].indexOf(s) >= 0)
```

Comment:
Make this array a variable to avoid repetition 
Use `Array.includes` instead of `indexOf`